src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,31 @@
+package core.basesyntax;
+
+public abstract class  Figure {
```

Comment:
```suggestion
public abstract class Figure implements AreaCalculator {
```
We also need to create another Drawable interface with the draw () method, which we will implement the figure, respectively, in each figure we will need to write its implementation